src/pipes/sky-date/sky-date.ts

```diff
@@ -0,0 +1,57 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+/**
+ * Generated class for the SkyDatePipe pipe.
+ *
+ * See https://angular.io/docs/ts/latest/guide/pipes.html for more info on
+ * Angular Pipes.
+ */
+@Pipe({
+  name: 'skydate',
+})
+export class SkyDatePipe implements PipeTransform {
+  /**
+   * Takes a value and makes it transform like the design.
+   */
+  transform(value: string) {
+    let _value: string = '';
```

Comment:
dont use let for vars that are not reassigned
dont type hint when its automatically deduced
dont prefix protected vars with _, use private scope for properties tho